OKR.md

```diff
@@ -2,6 +2,27 @@
 
 We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
+## 2019 Q1
+### Connectivity Magic
+* Gossipsub interops with go
+* Rendezvous Protocol is implemented
+* Connection manager supports dial backoff
+* QUIC transport in js
+* POC of WebBluetooth
+### Improved Performance
+* multistream 2.0 is implemented
+* DHT has stream support for searches
+* Improve DHT scalability
```

Comment:
For improving the DHT scalability we need @jhiesey feedback. @jhiesey do you have any idea of how long it will take to get the spec for the new DHT?